release.sh

```diff
@@ -204,14 +202,27 @@ else
     git commit -n -m "v$VERSION"
     git diff --stat --exit-code # check for uncommitted changes
     git tag -a "v$VERSION" -m "v$VERSION"
-    git push origin "v$VERSION"
+    git push "$ORIGIN" "v$VERSION"
 
     # Update npm (cdnjs update automatically)
-    yarn publish --new-version "${VERSION}"
+    # Fallback to npm publish, if yarn cannot authenticate, e.g., 2FA
+    yarn publish --new-version "$VERSION" || npm publish
 
     # Publish the website
+    # If gh-pages branch is protected, push to another branch
     pushd website
-    USE_SSH=true yarn publish-gh-pages
+    PUBLISH_GH_PAGES=$(USE_SSH=true yarn publish-gh-pages 2>&1 | tee /dev/tty || true)
```

Comment:
Will `yarn publish-gh-pages` respect `ORIGIN`?

Reply:
@edemaine It will clone GitHub Pages repository fresh, which is Khan/KaTeX for KaTeX, and therefore it is set to `origin`.